api/apps/api/src/modules/scenarios/scenarios.service.ts

```diff
@@ -118,6 +118,7 @@ import {
 } from './delete-scenario/delete-scenario.command';
 import { LegacyProjectImportChecker } from '../legacy-project-import/domain/legacy-project-import-checker/legacy-project-import-checker.service';
 import { lastValueFrom } from 'rxjs';
+import { AdjustPlanningUnitsInput } from '@marxan-api/modules/analysis/entry-points/adjust-planning-units-input';
```

Comment:
note to self: one knows that a module is way too big and broad when imports run down to line 121 😢